src/SpeedDial/SpeedDial.js

```diff
@@ -0,0 +1,198 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import { findDOMNode } from 'react-dom';
+import keycode from 'keycode';
+import { withStyles } from '../styles';
+import Button from '../Button';
+import Icon from '../Icon';
+import AddIcon from '../svg-icons/Add';
+
+const styles = theme => ({
+  root: {
+    width: 56,
+    textAlign: 'center',
+    zIndex: theme.zIndex.speedDial,
+    display: 'flex',
+    flexDirection: 'column-reverse', // Place the Actions above the FAB.
+  },
+  fab: {
+    transition: theme.transitions.create('transform', {
+      duration: theme.transitions.duration.shortest,
+    }),
+  },
+  hiddenFab: {
+    transform: 'scale3d(0, 0, 1)',
```

Comment:
Why using scale 3d and not?
```jsx
transform: 'scale(0)',
```

Reply:
Bacause I used scale, and you asked why I wasn't using scale3d. LOL!